docs/ocis/development/build.md

```diff
@@ -22,15 +22,23 @@ cd ocis
 
 ## Build the oCIS binary
 
-The oCIS binary source is in the `ocis` folder inside the oCIS repository. In this folder you can build the oCIS binary:
+Only if you have changed protobuf definitions or the frontend part in one of the extensions, you need to run following command in the root directory of the repository. Otherwise you can skip this step and proceed building the oCIS binary.
+This will at least modify multiple `embed.go` files because we embed the frontend build output in these `embed.go` files and a timestamp will be updated and also minor differences are expected between different Node.js versions.
```

Comment:
```suggestion
This will usually modify multiple `embed.go` files because we embed the frontend build output in these `embed.go` files and a timestamp will be updated and also minor differences are expected between different Node.js versions.
```